src/main/java/org/reaktivity/nukleus/tls/internal/stream/TlsServerFactoryBuilder.java

```diff
@@ -49,7 +49,7 @@
 
     public TlsServerFactoryBuilder(
         TlsConfiguration config,
-        Function<String, TlsStoreInfo> lookupStore)
+        Function<Long, TlsStoreInfo> lookupStore)
```

Comment:
`lookupStore` should be of type `LongFunction<TlsStoreInfo>` to avoid primitive auto-boxing of `long` to `Long` when called.